sklearn/tests/test_calibration.py

```diff
@@ -119,12 +119,7 @@ def test_sample_weight_warning():
     for method in ['sigmoid', 'isotonic']:
         base_estimator = LinearSVC(random_state=42)
         calibrated_clf = CalibratedClassifierCV(base_estimator, method=method)
-        # LinearSVC does not currently support sample weights but they
```

Comment:
Should we remove this test alltogether? It looks like it was added to test a warning that is not issued anymore. If it is still useful we should rename the test function to better reflect what it is testing.